weblate/vendasta/access.py

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+from weblate.auth.models import Group
+
+
+def set_permissions(strategy, backend, user, details, **kwargs):
+    """
+    Add all users to Viewers, add developers to 'Vendasta Internal', and add namespaced users to namespace group.
+    """
+    groups_to_add = [Group.objects.get('Viewers')]
+
+    roles = details.get('roles', [])
+    if 'developer' in roles:
+        groups_to_add.append(Group.objects.get(name='Vendasta Internal'))
```

Comment:
Should the group name be 'Vendasta Developers' so it is clear it does not include all employees of Vendasta?

Reply:
Makes sense! I think I initially meant for this to be any vendastian, but I recognized that didn't necessarily track.